link-grammar/string-id.h

```diff
@@ -0,0 +1,39 @@
+/*************************************************************************/
+/* Copyright (c) 2004                                                    */
+/* Daniel Sleator, David Temperley, and John Lafferty                    */
+/* All rights reserved                                                   */
```

Comment:
You should remove these three lines, and put in your name, instead. I'm not sure, but the I think the "all rights reserved" line is not necessary.

Reply:
Oh, I see, I guess you are thinking that this is a "minor" change to the older existing string-set code.  In that case, you can leave their copyright in place; but you should add your own name, as it is a fairly major revision.